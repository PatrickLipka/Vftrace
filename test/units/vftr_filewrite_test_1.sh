#!/bin/bash
set -x
ref_out_dir=ref_output
ref_in_dir=ref_input
testname=vftr_filewrite_test_1
outfile=$testname.out

rm -f $outfile
echo $HAS_MPI

if [ "x$HAS_MPI" == "xYES" ]; then
   $MPI_EXEC $NP 1 ./test_vftrace $testname
else
   ./test_vftrace $testname
fi
diff $ref_out_dir/$outfile $outfile

