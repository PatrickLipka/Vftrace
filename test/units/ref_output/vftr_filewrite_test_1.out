Check the creation of log and vfd file name
logfile_name(0, 1): ./test_vftrace_0.log
logfile_name(11, 111): ./test_vftrace_011.log
logfile_name(11, 111): ./test_vftrace_011.vfd
